lib/kube/proxy/resource_filters.go

```diff
@@ -173,10 +178,10 @@ func (d *resourceFilterer) FilterObj(obj runtime.Object) (isAllowed bool, isList
 	case *corev1.Pod:
 		result, err := filterResource(d.kind, d.verb, o, d.allowedResources, d.deniedResources)
 		if err != nil {
-			d.log.WithError(err).Warn("Unable to compile role kubernetes_resources.")
+			d.log.WithError(err).Warn("Unable to compile regex expressions within kubernetes_resources.")
```

Comment:
OK, now I understand where the error message comes from, but I still think we should just say something along "unable to filter resources", or something. This level of specificity comes from cases found four levels deeper in the call tree, and there's no guarantee that further changes will not introduce another error cases. In fact, they are already here, for example [this one](https://github.com/gravitational/teleport/blob/32ccd98af00120960ee23090a6f5f1e7c302245a/lib/utils/replace.go#L149). If we really encounter it in production and will see something about inability to compile regular expressions, this will only add to confusion.

Note that I'm assuming that `WithError(err)` adds the original error as context, so that even if we put a generic message here, the original error will still tell us what was wrong specifically.